tests/checkBoxes.spec.ts

```diff
@@ -1,80 +1,64 @@
 import { test, expect } from '@playwright/test';
 import { log } from 'console';
 import exp from 'constants';
-//1. Select the VETERINARIANS menu item in the navigation bar, then select "All"
-//2. Add assertion of the "Veterinarians" text displayed above the table with the list of Veterinarians
 
 test.beforeEach(async ({ page }) => {
+    //1. Select the VETERINARIANS menu item in the navigation bar, then select "All"
     await page.goto('/')
     await page.getByText('Veterinarians').click()
     await page.getByText('All').click()
-    await expect(page.getByRole("heading")).toHaveText("Veterinarians")
 })
 
-//3. Select the first veterinarian "Helen Leary" and click "Edit Vet" button
 test('Validate selected specialities', async ({ page }) => {
-    await page.getByText('Edit Vet').nth(1).click()
-    //4. Add assertion of the "Specialties" field. The value "radiology" is displayed
-    // Open the dropdown to make the option visible
-    const expandDropdownForCheckboxes = await page.locator('.dropdown-display').click();
-
-    // Wait for the dropdown content to be visible
-    const waitForDropdownVisibility = await page.waitForSelector('.dropdown-content');
-
-    // Optionally, check the dropdown content
-    const dropdownContent = await page.locator('.dropdown-content');
-    await expect(dropdownContent).toContainText('radiology');
-
-    // Simulate checking the checkbox if needed
-    //create const for checkboxes 
-    const cardiologyCheckbox = await page.locator('#radiology')
-    const dentistryCheckbox = await page.locator('#dentistry')
-    const surgeryCheckbox = await page.locator('#surgery')
-    await (cardiologyCheckbox).check();
+    //2. Add assertion of the "Veterinarians" text displayed above the table with the list of Veterinarians
+    await expect(page.getByRole("heading")).toHaveText("Veterinarians")
 
-    // Wait for the selected specialties element to be visible
-    await page.waitForSelector('.selected-specialties');
+    //3. Select the first veterinarian "Helen Leary" and click "Edit Vet" button
+    //Clicks seconds vet from the list
+    const edit_Vet_Button = page.getByText('Edit Vet')
+    edit_Vet_Button.nth(1).click()
```

Comment:
`edit_Vet_Button` not used later in the code, so not needed. can be as a single line command